scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -45,6 +45,9 @@ SG_ Frequency : 32|32@1- (1,0) [-3.4E038|3.4E038] "Hz" LOGGER
 BO_ 107 BMS_STATE_MACHINE : 1 BMS
 SG_ State : 0|8@1+ (1,0) [0|255] "" LOGGER
 
+BO_ 108 BMS_STATE_OF_CHARGE: 4 BMS
```

Comment:
Not strictly related to this PR, but we definitely want to display SoC on 7-seg at some point by reading the CAN bus. So I thought why not add this in.